ports/normancoresdk/portfile.cmake

```
@@ -0,0 +1,16 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO norman-ml/norman_core_sdk_cpp
+    REF v0.0.1

```

The port version is 1.0.0 but this REF is 0.0.1?

With proper sync, this could be written as
```suggestion
    REF v${VERSION}
```